app/models/location.rb

```diff
@@ -4,6 +4,8 @@ class Location < ActiveRecord::Base
   validates :name, presence: true
   validates :address, presence: true
 
+  scope :confirmed_by_gi, ->(confirmed_by_gi) { where(confirmed_by_gi: confirmed_by_gi) }
```

Comment:
I think it would be better to have this scope only be called when you care about filtering down to `confirmed_by_gi` records.It will read more naturally when invoking `Location.confirmed_by_gi`. So the code would end up looking something like:

`scope :confirmed_by_gi, -> { where(confirmed_by_gi: true) }`